HacktoberfestProject.Web/Services/GithubService.cs

```diff
@@ -29,10 +29,30 @@ public async Task<List<Models.DTOs.Repository>> GetRepos(string owner)
 
         public async Task<List<Models.DTOs.PullRequest>> GetPullRequestsForRepo(string owner, string name)
         {
-            _logger.LogTrace($"Sending request to Github for pull requests on repositoy: {name}");
+            _logger.LogTrace($"Sending request to Github for pull requests on repository: {name}");
             var prs = await _client.PullRequest.GetAllForRepository(owner, name, new PullRequestRequest() { State = ItemStateFilter.All });
 
             return prs.Select(pr => new Models.DTOs.PullRequest(pr.Number, pr.Url)).ToList();
         }
+
+        public async Task<IEnumerable<string>> SearchOwners(string owner)
+        {
+            var searchResults = new List<string>();
+
+            if (!string.IsNullOrWhiteSpace(owner))
+            {
+                _logger.LogTrace($"Sending request to Github for owners like: {owner}");
+                var users = await _client.Search.SearchUsers(new SearchUsersRequest(owner)
+                {
+                    AccountType = AccountSearchType.User,
+                    In = new[] { UserInQualifier.Username }
+                });
+
+                NullChecker.IsNotNull(users, nameof(users));
```

Comment:
I think using the null checker here may not be the right thing.
In a constructor, if a dependency is null, then that is exceptional. It also throws a ` new ArgumentNullException(argumentName)` which isn't correct in this context.
If a `users` comes back null we want to tell our user that there was an error, perhaps a typo. Not throw an exception. Does that make sense?